pkg/cluster/k8sres.go

```diff
@@ -99,15 +99,15 @@ func (c *Cluster) serviceAddress(role PostgresRole) string {
 	return ""
 }
 
-func (c *Cluster) servicePort(role PostgresRole) string {
+func (c *Cluster) servicePort(role PostgresRole) int32 {
```

Comment:
that is better renamed `generateServicePort` - otherwise the logic of returning `5432` when a service does not exist is vauge

Reply:
Yes something like `getServicePort` would be better. But, now it's also in line with the other functions like `serviceName`, `serviceAddress`. I think, I would keep the name.